src/App/App.js

```diff
@@ -23,8 +25,16 @@ class App extends Component {
       .catch(error => this.setState({ error: error.message }));
   }
 
-  returnToHome = () => {
-    this.setState({ movieDetails: null })
+  getMoviesByRating = (ratingMinMax) => {
+    const range = ratingMinMax.split(","); 
+    const moviesInRange = this.state.movies.filter(movie => {
+      return movie.average_rating >= +range[0] && movie.average_rating <= +range[1]
+    })
+    if (moviesInRange.length === 0) {
+      this.setState({moviesByRating: ["NaN"]})
```

Comment:
I like that this "NaN" worked. I wonder if there is a better way to do this. Let me know if you find a better one!